core/java/iesi-core/src/main/java/io/metadew/iesi/connection/database/postgresql/PostgresqlDatabaseConnectionService.java

```diff
@@ -27,4 +27,9 @@ public String getDriver(PostgresqlDatabaseConnection databaseConnection) {
     public Class<PostgresqlDatabaseConnection> appliesTo() {
         return PostgresqlDatabaseConnection.class;
     }
+
+    @Override
+    public String refactorLimitAndOffset(PostgresqlDatabaseConnection databaseConnection, String query) {
+        return query;
+    }
```

Comment:
All XDatabaseConnectionService's extend either DatabaseConnectionService or SchemaDatabaseConnectionService (which extends DatabaseConnectionService). You could implement the default behavior in the DatabaseConnectionService class